
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://mixs-minas.org/documentation/project-documentation/example-sample-data-description-use-cases/">
      
      
        <link rel="prev" href="../feedback-session-protocol/">
      
      
        <link rel="next" href="../../../contributors/">
      
      
      <link rel="icon" href="../../../assets/images/logos/minas-favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.21">
    
    
      
        <title>Example use cases - MIxS-MInAS</title>
      
    
    
      <link rel="stylesheet" href="../../../assets/stylesheets/main.2a3383ac.min.css">
      
        
        <link rel="stylesheet" href="../../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Albert+Sans:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Albert Sans";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#example-sample-and-data-descriptions-use-cases" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../.." title="MIxS-MInAS" class="md-header__button md-logo" aria-label="MIxS-MInAS" data-md-component="logo">
      
  <img src="../../../assets/images/logos/minas-logo-rectangle-monochrome.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS-MInAS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Example use cases
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme)" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m14.3 16-.7-2h-3.2l-.7 2H7.8L11 7h2l3.2 9zM20 8.69V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12zm-9.15 3.96h2.3L12 9z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_2" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to system preference"  type="radio" name="__palette" id="__palette_2">
    
      <label class="md-header__button md-icon" title="Switch to system preference" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/MIxS-MInAS/MIxS-MInAS.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../.." title="MIxS-MInAS" class="md-nav__button md-logo" aria-label="MIxS-MInAS" data-md-component="logo">
      
  <img src="../../../assets/images/logos/minas-logo-rectangle-monochrome.png" alt="logo">

    </a>
    MIxS-MInAS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/MIxS-MInAS/MIxS-MInAS.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    About
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            About
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../about/proposal/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Proposal
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../about/phases/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Project Phases
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <div class="md-nav__link md-nav__container">
            <a href="../../../schema/" class="md-nav__link ">
              
  
  
  <span class="md-ellipsis">
    Drafts
    
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_3" id="__nav_3_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Drafts
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../schema/selecter/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MInAS checklist selection help
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../schema/combinations/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MInAS submission combinations
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../schema/ancient-extension/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Ancient extension
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../schema/radiocarbon-dating-extension/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Radiocarbon dating extension
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../schema/existing-term-updates/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MIxS term updates
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../schema/draft/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Deprecated
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" checked>
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Documentation
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Documentation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_1" >
        
          
          <label class="md-nav__link" for="__nav_4_1" id="__nav_4_1_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    User Documentation
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_1">
            <span class="md-nav__icon md-icon"></span>
            User Documentation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../user-documentation/overview/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../user-documentation/faq/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    FAQ
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_2" >
        
          
          <label class="md-nav__link" for="__nav_4_2" id="__nav_4_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Developer Documentation
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_2">
            <span class="md-nav__icon md-icon"></span>
            Developer Documentation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../developer-documentation/overview/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
    
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_3" checked>
        
          
          <label class="md-nav__link" for="__nav_4_3" id="__nav_4_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Project Documentation
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4_3">
            <span class="md-nav__icon md-icon"></span>
            Project Documentation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../overview/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../contributing/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Contributing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../feedback-session-protocol/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Running feedback sessions
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Example use cases
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Example use cases
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#introduction" class="md-nav__link">
    <span class="md-ellipsis">
      Introduction
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-1" class="md-nav__link">
    <span class="md-ellipsis">
      Example 1
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-2" class="md-nav__link">
    <span class="md-ellipsis">
      Example 2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-3" class="md-nav__link">
    <span class="md-ellipsis">
      Example 3
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-4" class="md-nav__link">
    <span class="md-ellipsis">
      Example 4
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-5" class="md-nav__link">
    <span class="md-ellipsis">
      Example 5
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-6" class="md-nav__link">
    <span class="md-ellipsis">
      Example 6
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-7" class="md-nav__link">
    <span class="md-ellipsis">
      Example 7
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-8" class="md-nav__link">
    <span class="md-ellipsis">
      Example 8
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-9" class="md-nav__link">
    <span class="md-ellipsis">
      Example 9
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../contributors/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Contributors
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../contact/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Getting Involved
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../code-of-conduct/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Code of Conduct
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#introduction" class="md-nav__link">
    <span class="md-ellipsis">
      Introduction
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-1" class="md-nav__link">
    <span class="md-ellipsis">
      Example 1
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-2" class="md-nav__link">
    <span class="md-ellipsis">
      Example 2
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-3" class="md-nav__link">
    <span class="md-ellipsis">
      Example 3
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-4" class="md-nav__link">
    <span class="md-ellipsis">
      Example 4
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-5" class="md-nav__link">
    <span class="md-ellipsis">
      Example 5
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-6" class="md-nav__link">
    <span class="md-ellipsis">
      Example 6
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-7" class="md-nav__link">
    <span class="md-ellipsis">
      Example 7
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-8" class="md-nav__link">
    <span class="md-ellipsis">
      Example 8
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-9" class="md-nav__link">
    <span class="md-ellipsis">
      Example 9
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="example-sample-and-data-descriptions-use-cases">Example Sample and Data Descriptions Use Cases</h1>
<h2 id="introduction">Introduction</h2>
<p>This document aims to serve as reference guidance for people who are participating in feedback sessions, and as useful information for colleagues at the ENA sequencing data archive, who want to get better insights into how ancient DNA researchers submit their data.</p>
<p>As preparation for testing the MInAS checklist, we recommend users try and write out a <em>minimal</em> description for the <em>data output</em> of one or two samples in ‘materials and methods’ - like prose. This should include all the information they would normally put in their own publications, or more importantly, what they would like to see other ancient DNA researchers report.</p>
<p>These descriptions can be of:</p>
<ul>
<li>their own unpublished samples/data (anonymised if necessary)</li>
<li>fake ‘dummy’ samples</li>
<li>previously published samples and data</li>
</ul>
<p>For the latter case, we would request that the participant also include the ENA/NCBI/DDBJ accession number(s) of their uploaded data (or similar database), so the ENA can compare what the participant has written with what they actually uploaded to the data archiving repositories (pre-MInAS).</p>
<h2 id="example-1">Example 1</h2>
<p>A 1000 year old rat bone from an archaeological site is sampled to generate a non-UDG treated metagenomic library. Library is shotgun sequenced, generating a metagenomic sequencing file. The rat DNA in this file mapped to the rat genome, resulting in a reconstructed ancient rat genome that is published in Publication A. Later, the same library is capture-enriched for the <em>Yersinia pestis</em> pathogen genome and then sequenced, resulting in a new sequencing file. The data from this file is then aligned to the Yersinia pestis genome, resulting in a reconstructed ancient <em>Yersinia pestis</em> genome that is published in Publication B.</p>
<p>Summary</p>
<ul>
<li>Sample: Rat bone</li>
<li>Libraries:<ul>
<li>Metagenomic library</li>
<li>Post-capture <em>Y. pestis</em> pathogen-enriched library</li>
</ul>
</li>
<li>Sequencing files:<ul>
<li>Raw data of shotgun-sequenced metagenomic library [fastq]</li>
<li>Raw data of sequenced post-capture <em>Y. pestis</em> pathogen enriched library [fastq]</li>
</ul>
</li>
<li>Genome/Assembly files<ul>
<li>1 reference-aligned ‘genome’ of rat [bam]</li>
<li>1 reference-aligned ‘genome’ of <em>Y. pestis</em> [bam]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Unpublished</li>
</ul>
<h2 id="example-2">Example 2</h2>
<p>Residue from a medieval manuscript is sampled to generate a metagenomic library. The library is then shotgun sequenced, generating a metagenomic sequencing file. The resulting metagenomic data is then <em>de novo</em> assembled, resulting in 10 MAGs of mostly environmental bacterial taxa. Later the library is then capture-enriched for a panel of 10 skin bacteria, resulting in a sequencing file containing partial genomes of 3 skin microbial genomes.</p>
<p>Summary</p>
<ul>
<li>Sample: Medieval manuscript</li>
<li>Libraries:<ul>
<li>Metagenomic library</li>
<li>Post-capture skin bacteria-enriched library</li>
</ul>
</li>
<li>Sequencing files<ul>
<li>Raw data of shotgun sequenced metagenomic library [fastq]</li>
<li>Raw data of sequenced post-capture skin bacteria-enriched library [fastq]</li>
</ul>
</li>
<li>Genome/Assembly files<ul>
<li>10 MAGs of environmental bacterial taxa [fasta]</li>
<li>3 reference-aligned genomes of skin bacteria [bam]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Unpublished</li>
</ul>
<h2 id="example-3">Example 3</h2>
<p>Dental calculus was sampled from an adult historical chimpanzee (<em>Pan troglodytes verus</em>) skeleton (biological sex unknown) from an early 20th century museum collection (museum ID: SMF-PA-PC-071). The exact location of where the chimpanzee originated is unknown, but it was collected by early 20th century adventurers traveling in Nimba Liberia, sometime around 1930. Calculus deposits from multiple teeth were pooled into a single sample tube for analysis (lab code: CDC005.A).</p>
<p>The amount calculus was not sufficient for weighing on standard scales. No museum glue was observed on the specimen, so glue removal protocols were not performed. The sample was decontaminated by UV irradiation prior to analysis. DNA was extracted (qPCR concentration: 941,500 copies/µl) and built into a non-UDG treated library (Lab ID: CDC005.A0101) using the single-stranded DNA library protocol (Gansauge and Meyer 2013). The library was sequenced in two separate runs on two different instruments: first on a HiSeq4000 to assess data quality (20M reads, 75 bp single-end sequencing, SR), and later on a NextSeq 500 to produce a large dataset (100M read pairs, 75 bp paired-end sequence, PE).</p>
<p>Reads were trimmed with AdapterRemoval to remove adapter sequences and reads &lt;20 bp in length. Paired-end reads were not merged for uploading. The resulting data was analyzed to determine the microbial community structure of chimpanzee dental calculus.</p>
<p>Summary</p>
<ul>
<li>Sample: Dental calculus (microbial biofilm) collected from chimpanzee (host) teeth</li>
<li>Libraries:<ul>
<li>Metagenomic library</li>
</ul>
</li>
<li>Sequencing files<ul>
<li>Raw data of shotgun sequenced metagenomic library (HiSeq, single-end) [fastq]</li>
<li>Raw data of shotgun sequenced metagenomic library (NextSeq, paired-end) [fastq]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Publication: <a href="https://doi.org/10.1073/pnas.2021655118">https://doi.org/10.1073/pnas.2021655118</a></li>
<li>ENA study accession: PRJEB34569</li>
<li>ENA sample accession: ERS3774412 (SAMEA5971569)</li>
<li>ENA run accessions: ERR3579671; ERR3678173</li>
<li>Contact: James Fellows Yates (james_fellows_yates@eva.mpg.de)</li>
</ul>
<h2 id="example-4">Example 4</h2>
<p>Dental calculus was sampled from a Neanderthal (<em>Homo neanderthalensis</em>) first molar (Collection ID: Pes-3, Layer 4b; Lab ID: PES001.B) from Pešturina Cave in Serbia (Lat: 43.166667, Lon: 21.9). The age and biological sex of the individual is unknown, but the tooth has been Electron Spin Resonance (ESR) dated to 102,400 BP, and the sample is archived at the Department of Archaeology at University of Belgrade, Serbia. The tooth was excavated from Pešturina Cave during excavations conducted between 2006-2012, and the calculus sampled at the University of Belgrade in 2016. The sample was sent to Germany for analysis, and DNA extracted in 2016.</p>
<p>The calculus sample weighed 0.7mg, and underwent two steps of decontamination: UV radiation and an EDTA wash prior to extraction. DNA was extracted (qPCR concentration: 10,970,000 copies/µl). The extracted DNA was split into two aliquots; one aliquot was subjected to UDG treatment to remove ancient DNA damage in the form of uracils (full-UDG), and the other aliquot was untreated (non-UDG). Both aliquots were then built into standard Illumina DNA libraries. The non-UDG library was shallowly sequenced on a HiSeq4000 to assess quality and ancient DNA damage levels (75 bp single-end sequencing, SR; total reads: 12,350,535; Lab ID: PES001.B0101). The full-UDG library was deeply sequenced to produce a large dataset (75 bp paired-end sequencing, PE; total reads: 128,930,050; Lab ID: PES001.B0103).</p>
<p>For both sequencing files, reads were trimmed with AdapterRemoval to remove adapter sequences and reads \&lt;20 bp in length. Paired-end reads were not merged for uploading. The resulting data was analyzed to determine the microbial community structure of Neanderthal dental calculus.</p>
<p>Summary</p>
<ul>
<li>Sample: Dental calculus (microbial biofilm) from a Neanderthal (host) tooth</li>
<li>Libraries:<ul>
<li>Metagenomic library (non-UDG treated)</li>
<li>Metagenomic library (full-UDG treated)</li>
</ul>
</li>
<li>Sequencing files<ul>
<li>Raw data of shotgun sequenced non-UDG-treated metagenomic library (HiSeq, single-end) [fastq]</li>
<li>Raw data of shotgun sequenced full-UDG-treated metagenomic library (HiSeq, paired-end) [fastq]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Publication: <a href="https://doi.org/10.1073/pnas.2021655118">https://doi.org/10.1073/pnas.2021655118</a></li>
<li>ENA study accession: PRJEB34569</li>
<li>ENA sample accession: ERS3774441 (SAMEA5971598)</li>
<li>ENA run accessions: ERR3579688, ERR3678187</li>
<li>Contact: James Fellows Yates (james_fellows_yates@eva.mpg.de)</li>
</ul>
<h2 id="example-5">Example 5</h2>
<p>Sediment samples were collected from the Kap København Formation in North Greenland (Lat: 82.4, Lon: -23.3), a High Arctic deposit dating to approximately 2.0 million years ago. The sediment samples were in profiles at different locations and stored at the Centre for GeoGenetics, University of Copenhagen. DNA extraction and sequencing were conducted in Denmark, using a combination of targeted enrichment and shotgun sequencing approaches to reconstruct ancient ecosystems.<br />
A total of 41 sediment samples were extracted for DNA, yielding 65 dual-indexed Illumina sequencing libraries. Of these, 34 libraries underwent targeted enrichment for mammalian mitochondrial DNA using the PaleoChip Arctic1.0 bait set. All libraries were sequenced on an Illumina HiSeq 4000 (80 bp paired-end, PE) or NovaSeq 6000 (100 bp PE). A total of 16,882,114,068 reads were generated. After low-complexity filtering (Dust = 1), quality trimming (q ≥ 25), duplicate removal, and length filtering (≥ 30 bp for NovaSeq), 2,873,998,429 reads remained for downstream analysis.</p>
<p>Summary</p>
<ul>
<li>Sample: Sediment from the Kap København Formation (~2.0 million years old, Arctic ecosystem eDNA)</li>
<li>Libraries:<ul>
<li>Metagenomic library (non-UDG treated)</li>
<li>Target-enriched library (mammalian mitochondrial DNA, PaleoChip Arctic1.0)</li>
</ul>
</li>
<li>Sequencing files:<ul>
<li>Raw data of shotgun-sequenced non-UDG-treated metagenomic library (HiSeq 4000/NovaSeq 6000, paired-end) [fastq]</li>
<li>Raw data of targeted enrichment-sequenced mitochondrial DNA library (HiSeq 4000/NovaSeq 6000, paired-end) [fastq]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Publication: <a href="https://doi.org/10.1038/s41586-022-05453-y">https://doi.org/10.1038/s41586-022-05453-y</a></li>
<li>ENA study accession: PRJEB55522</li>
<li>Contact: Anonymous (not provided)</li>
</ul>
<h2 id="example-6">Example 6</h2>
<p>Sediment samples were collected in 2019 at the Velký Mamuťák rock shelter in northern Bohemia, Czech Republic. This site has been a forested enclave since the early Holocene. A notable shift in dominant grazing species was observed, transitioning from sheep during the Late Neolithic to pigs in the Bronze Age (approximately 4,100–3,000 years before present). In 2019, 28 sediment samples (~10 grams each) were systematically collected from the stratigraphic profile of the north wall of the excavation trench at Velký Mamuťák. To minimize contamination, the exposed layers were removed before sampling. The material was then transferred into sterile 15-mL centrifuge tubes using disposable scalpels, with researchers wearing face masks and nitrile gloves throughout the process. The samples were transported to the ancient DNA dedicated laboratories at the Globe Institute, University of Copenhagen, Denmark, and stored at -20°C until DNA extraction and library preparation. Sequenced reads underwent adapter trimming using AdapterRemoval and were processed through the 'Holi' pipeline for quality control, low complexity filtering, and dereplication.</p>
<p>Summary</p>
<ul>
<li>Sample: Sediment from Velký Mamuťák rock shelter, northern Bohemia, Czech Republic (Late Neolithic to Bronze Age)</li>
<li>Libraries:<ul>
<li>Metagenomic library</li>
</ul>
</li>
<li>Sequencing Files:<ul>
<li>Raw data of shotgun-sequenced metagenomic library (paired-end) [fastq]</li>
</ul>
</li>
</ul>
<p>Details:</p>
<ul>
<li>Publication: <a href="https://doi.org/10.1016/j.cub.2024.10.062">https://doi.org/10.1016/j.cub.2024.10.062</a></li>
<li>ENA Study Accession: PRJEB59830</li>
<li>Contact: ?????</li>
</ul>
<h2 id="example-7">Example 7</h2>
<p>A ~1000 year old human bone from an archaeological site is sampled to generate a non-UDG treated library. Library is shotgun sequenced, generating a first sequencing file. The human DNA mapped to the human (hg37) genome, resulting in a reconstructed ancient human genome. After quality control, the same library was selected for hybridization capture enrichment, targeting ~1.2 million single-nucleotide polymorphisms (SNPs) genome wide (1240k SNP capture) and then sequenced, resulting in a new sequencing file. The data from this file is again aligned to the human (hg37) genome. After, to increase the number of analysable sites, the library was enriched again for a new capture that increases the SNP target number to ~1.4 million (twist bioscience capture), resulting in another sequence, also mapped to the human (hg37) genome. Lastly, the library was submitted to a mitochondrial DNA capture designed to target mitochondrial reads, which was then sequenced and aligned to the mitochondrial human (hg37) reference genome.</p>
<p>Summary</p>
<ul>
<li>Sample: Human bone</li>
<li>Libraries:<ul>
<li>Shotgun sequence library</li>
<li>Capture sequence library (1240K)</li>
<li>Capture sequence library (twist)</li>
<li>Capture sequence library (MT)</li>
</ul>
</li>
<li>Sequencing files:<ul>
<li>Raw data of shotgun sequence human library [fastq]</li>
<li>Raw data of capture (1240K) sequence library [fastq]</li>
<li>Raw data of capture (twist) sequence library [fastq]</li>
<li>Raw data of capture (MT) sequence library [fastq]</li>
</ul>
</li>
<li>Genome/Assembly files<ul>
<li>Shotgun sequence aligned to the human genome [bam]</li>
<li>Capture(1240K) sequence aligned to the human genome [bam]</li>
<li>Capture(twist) sequence aligned to the human genome [bam]</li>
<li>Capture(MT) sequence aligned to the human mitochondrial genome [bam]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Unpublished</li>
<li>Contact: Eleni Seferidou (eleni_seferidou@eva.mpg.de)</li>
</ul>
<h2 id="example-8">Example 8</h2>
<p>Thirty-nine woolly mammoth (<em>Mammuthus primigenius</em>) teeth and bones ~24,000 years old were sampled from the archaeological site of Kostenki 11-Ia in south-western Russia for ancient DNA (aDNA) analysis. Some of the DNA extracts were treated with Thermolabile USER II enzyme (NEB) prior to library build (i.e. they were UDG-treated), while others were not UDG-treated. Sequencing libraries were built using either a double stranded DNA (dsDNA) (Meyer and Kircher, 2010) or a single stranded DNA (ssDNA) (Kapp et al., 2021) method, but never both for the same sample. All the libraries were double-indexed. Indexed libraries were shotgun sequenced on an Illumina HiSeq 2500 SE80 base pairs (bp) or NovaSeq 6000 PE150 bp to generate the first set of sequencing files.</p>
<p>We performed genetic sex determination of the 23 mammoth specimens that had &gt;5000 reads. We estimated the ratio of the number of reads mapping to sex chromosome X versus an autosome (Chr8) of the African savannah elephant. [Relevant as the bam files were uploaded to the SRA].</p>
<p>To increase the number of mitochondrial sequences, we performed mitochondrial enrichment by target capture on all the ssDNA libraries using the myBaits v5.01 High Sensitivity protocol. A custom capture panel containing baits specific to the mitochondrial genome of <em>M. primigenius</em> was used. Post-capture amplification was performed using NEBNext Q5U Master Mix (NEB) as follows: 98℃ for 2min, 18 cycles of 98℃ for 20s, 60℃ for 30s, and 72℃ for 45s, with a final extension step of 72℃ for 5min. The captured libraries were pooled equimolar and sequenced on an Illumina NovaSeq X Plus PE150 bp.</p>
<p>The assembled mitogenomes from the shotgun (n=2) and capture (n=14) data were uploaded to GenBank.</p>
<p>Summary</p>
<ul>
<li>Samples: Teeth and bones from woolly mammoth (<em>Mammuthus primigenius</em>)</li>
<li>Libraries:<ul>
<li>Double-stranded shotgun sequencing libraries (non-UDG treated)</li>
<li>Single-stranded shotgun sequencing libraries (full-UDG treated)</li>
<li>Target enriched (capture) libraries (ssDNA, full-UDG treated, mitochondrial)</li>
</ul>
</li>
<li>Sequencing files<ul>
<li>Raw data of shotgun sequenced non-UDG treated dsDNA libraries (HiSeq 2500 SE80) [fastq]</li>
<li>Raw data of shotgun sequenced full-UDG treated ssDNA libraries (NovaSeq 6000 PE150) [fastq]</li>
<li>Raw data of capture libraries (ssDNA, full-UDG treated, NovaSeq X Plus PE150) [fastq]</li>
</ul>
</li>
<li>Genome/Assembly files:<ul>
<li>Shotgun sequenced reads (dsDNA and ssDNA) mapped to ChrX of African savannah elephant [bams]</li>
<li>Shotgun sequenced reads (dsDNA and ssDNA) mapped to Chr8 of African savannah elephant [bams]</li>
<li>Assembled mitochondrial genomes (partial or complete) uploaded to GenBank [fasta]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Publication: <a href="https://doi.org/10.1016/j.qeh.2024.100049">https://doi.org/10.1016/j.qeh.2024.100049</a></li>
<li>ENA (SRA) study accession: PRJNA897816</li>
<li>ENA (SRA) sample accessions: SAMN44842740 (dsDNA lib, shotgun SE80), SAMN44842744 (ssDNA lib, shotgun PE150, capture PE150)</li>
<li>ENA (SRA) run accessions: SRR31414997 (dsDNA lib, shotgun SE80), SRR31414936 (ssDNA lib, shotgun PE150), SRR31414935 (ssDNA lib, capture PE150)</li>
<li>GenBank accessions: PQ631146 (assembled mitogenome example)</li>
<li>Contact: Deon de Jager (deon.dejager@sund.ku.dk / dejager4@gmail.com)</li>
</ul>
<h2 id="example-9">Example 9</h2>
<p>A total of 995 ancient chicken bones from 148 western Eurasian archaeological sites dating from the last 2,700 years were sampled for palaeogenomic analyses. During sample selection, emphasis was given to the coracoid due to its higher density when compared to long bones. In the absence of this element in a given assemblage, long bones were selected. Prior to grinding, ~0.5mm of the exterior surface of the bone was removed using a Dremel 3000 electric hand-drill. No bleaching of the bone powder was conducted prior to DNA extraction, which followed the Dabney protocol (Dabney et al., 2013). Non-UDG treated Illumina libraries were generated following the double stranded DNA (ds) protocols of either Meyer &amp; Kircher (2010) or Carøe et al. (2018). These double-indexed libraries were then pooled at equimolar levels and shotgun sequenced on an Illumina HiSeq 2500/4000/X (80-150 bp paired-end, PE). The resulting sequenced data was mapped against the <em>Gallus gallus</em> GRCg6a genome reference (GCF_000002315.6).</p>
<p>Based on the assessment of endogenous DNA in each sample, selected amplified libraries underwent enrichment for mitochondrial genomes using MYbaits (MYcroarray) with a 24-hour hybridisation at 55°C based on the MYbaits manual V3 (2016). These libraries were sequenced on an Illumina HiSeq 2500 (80bp single-end, SE) and mapped against the <em>Gallus gallus</em> GRCg6a mitochondrial genome reference (NC<em>040902.1). Meanwhile, libraries displaying higher endogenous DNA content were sequenced to greater depth for whole genome analysis on multiple Hi-Seq 2500 lanes (100bp SE). This newly-generated data was mapped against the _Gallus gallus</em> GRCg6a genome reference (GCF_000002315.6).</p>
<p>Marek's disease virus (MDV) is a highly contagious alphaherpesvirus that causes a tumor-associated disease in poultry. To investigate its evolutionary history, the shotgun sequenced libraries were screened for the <em>Gallid Alphaherpesvirus 2</em> (Marek Disease Virus type 1, MDV) genome using HAYSTAC (Dimopoulos et al., 2018 based on a custom database of complete herpesvirus genomes selected from the NCBI RefSeq database. Eighteen samples were found to present at least one MDV read. To validate the methodology, a feather sample from a modern chicken showing MDV symptoms, and a negative control consisting of an ancient sample with no MDV reads upon screening, were also included, totalling the number of samples to 20.</p>
<p>These 20 amplified ds libraries were subsequently enriched for viral DNA using a hybridisation-based capture approach based on RNA baits designed to cover the entire <em>Gallid alphaherpesvirus 2</em> genome (strain RB-1B; NCBI accession EF523390.1). These were then paired-end sequenced on an Illumina NovaSeq lane (150bp PE), generating MDV-specific sequencing files. Samples with uniquely mapped MDV reads of 25 bp or longer were identified as true positives for Marek’s disease, resulting in fifteen ancient genomes with coverage depths ranging from 0.13x to 41.92x. These were submitted to Genbank.</p>
<p>Summary</p>
<ul>
<li>Sample: Archaeological domestic chicken (<em>Gallus gallus</em>) bones</li>
<li>Libraries:<ul>
<li>Double-stranded, double-indexed shotgun sequencing libraries (non-UDG treated, Illumina)</li>
<li><em>Gallus gallus</em> mitochondrially-enriched (capture) libraries (non-UDG treated, Illumina)</li>
<li><em>Gallid alphaherpesvirus 2</em> genome enriched (capture) libraries (non-UDG treated, Illumina)</li>
</ul>
</li>
<li>Sequencing files<ul>
<li>Raw data of shotgun sequenced non-UDG treated ds libraries (HiSeq, PE)</li>
<li>Raw data of <em>Gallus gallus</em> mitochondrially-enriched (capture) non-UDG treated libraries (HiSeq, SE)</li>
<li>Raw data of <em>Gallid alphaherpesvirus 2</em> enriched (capture) non-UDG treated libraries (NovaSeq, PE)</li>
<li>Assembled mitochondrial genomes (partial or complete) of host <em>Gallus gallus</em> uploaded to GenBank [NOT PUBLISHED]</li>
<li>Assembled nuclear genomes (partial or complete) of host <em>Gallus gallus</em> uploaded to GenBank [NOT PUBLISHED]</li>
<li>Assembled genomes (partial or complete) of <em>Gallid alphaherpesvirus 2</em> uploaded to GenBank [NOT UPLOADED - THEORETICAL]</li>
</ul>
</li>
</ul>
<p>Details</p>
<ul>
<li>Host (chicken) data unpublished; description of mitochondrial enrichment and whole genome sequencing contain partial ‘dummy’ methodology</li>
<li>MDV (<em>Gallid Alphaherpesvirus 2</em>) Publication: <a href="https://doi/10.1126/science.adg2238">https://doi.org/10.1126/science.adg2238</a></li>
<li>Note that contrary to what is mentioned in this example, the assembled genomes of <em>Gallid alphaherpesvirus 2</em> were not uploaded to Genbank.</li>
<li>MDV ENA study accession: PRJEB64489</li>
<li>MDV ENA sample accession: ERS16164982 - ERS16165001 (SAMEA114184281 - SAMEA114184300)</li>
<li>MDV ENA run accessions: ERR11752787 - ERR11752836</li>
<li>Contact: Ophélie Lebrasseur (ophelie.lebrasseur@ird.fr)</li>
</ul>







  
    
  
  


  <aside class="md-source-file">
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Last update">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 13.1c-.1 0-.3.1-.4.2l-1 1 2.1 2.1 1-1c.2-.2.2-.6 0-.8l-1.3-1.3c-.1-.1-.2-.2-.4-.2m-1.9 1.8-6.1 6V23h2.1l6.1-6.1zM12.5 7v5.2l4 2.4-1 1L11 13V7zM11 21.9c-5.1-.5-9-4.8-9-9.9C2 6.5 6.5 2 12 2c5.3 0 9.6 4.1 10 9.3-.3-.1-.6-.2-1-.2s-.7.1-1 .2C19.6 7.2 16.2 4 12 4c-4.4 0-8 3.6-8 8 0 4.1 3.1 7.5 7.1 7.9l-.1.2z"/></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-iso_date" title="October 15, 2025 08:47:45 UTC">2025-10-15</span>
  </span>

    
    
    
    
  </aside>





                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
      <div class="md-consent" data-md-component="consent" id="__consent" hidden>
        <div class="md-consent__overlay"></div>
        <aside class="md-consent__inner">
          <form class="md-consent__form md-grid md-typeset" name="consent">
            



  

<h4>Cookie consent</h4>
<p>This website uses cookies to improve your experience. We'll assume you're ok with this, but you can opt-out if you wish</p>
<input class="md-toggle" type="checkbox" id="__settings" checked>
<div class="md-consent__settings">
  <ul class="task-list">
    
    
      
    
    
      
        
  
  
    
    
  
  <li class="task-list-item">
    <label class="task-list-control">
      <input type="checkbox" name="github" checked>
      <span class="task-list-indicator"></span>
      GitHub
    </label>
  </li>

      
    
    
  </ul>
</div>
<div class="md-consent__controls">
  
    
      <button class="md-button md-button--primary">Accept</button>
    
    
    
  
    
    
      <button type="reset" class="md-button md-button--primary">Reject</button>
    
    
  
</div>
          </form>
        </aside>
      </div>
      <script>var consent=__md_get("__consent");if(consent)for(var input of document.forms.consent.elements)input.name&&(input.checked=consent[input.name]||!1);else"file:"!==location.protocol&&setTimeout((function(){document.querySelector("[data-md-component=consent]").hidden=!1}),250);var form=document.forms.consent;for(var action of["submit","reset"])form.addEventListener(action,(function(e){if(e.preventDefault(),"reset"===e.type)for(var n of document.forms.consent.elements)n.name&&(n.checked=!1);__md_set("__consent",Object.fromEntries(Array.from(new FormData(form).keys()).map((function(e){return[e,!0]})))),location.hash="",location.reload()}))</script>
    
    
      
      <script id="__config" type="application/json">{"base": "../../..", "features": [], "search": "../../../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../../assets/javascripts/bundle.f55a23d4.min.js"></script>
      
        <script src="https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js"></script>
      
        <script src="../../../javascripts/tablesort.js"></script>
      
    
  </body>
</html>